camp/jbolotski/beam_break_motor/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME

```

Comment:
Will look for other .jar files (see above ^)

Reply:
I don't understand this comment. What change do you want made?

Reply:
just pointing out that there was a .jar file earlier, not sure if its still around ... its no big deal to have 1 jar file